### ISSUE TITLE
模型微调时报错，报内核版本问题 Detected kernel version 5.4.0

### ISSUE THREAD

**Issue by hgsw** [datawhalechina/self-llm#125]
在模型微调时，执行如下代码报错：
trainer = Trainer(
    model=model,
    args=args,
    train_dataset=tokenized_id,
    data_collator=DataCollatorForSeq2Seq(tokenizer=tokenizer, padding=True),
)
报错信息：
Detected kernel version 5.4.0, which is below the recommended minimum of 5.5.0; this can cause the process to hang. It is recommended to upgrade the kernel to the minimum version or higher.
有时不会不报错，有时就会出现。

**Comment by KMnO4-zx** [comment]
你用的是什么环境呀
